src/dotnet/CommandLine/CommandLineApplication.cs

```diff
@@ -129,10 +130,18 @@ public int Execute(params string[] args)
                     if (longOption != null)
                     {
                         processed = true;
-                        option = command.Options.SingleOrDefault(opt => string.Equals(opt.LongName, longOption[0], StringComparison.Ordinal));
+                        string longOptionName = longOption[0];
```

Comment:
I follow the [CoreFX coding guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md).  (See rule #10) And I think all C# code under the `dotnet` user should follow the same coding guidelines so people contributing to both don't have to switch their coding style in between repos.